pallets/settlement/src/lib.rs

```diff
@@ -1847,7 +1887,7 @@ impl<T: Config> Module<T> {
 
     /// Returns the worst case weight for an instruction with `f` fungible legs, `n` nfts being transferred and `o` offchain assets.
     fn execute_scheduled_instruction_weight_limit(f: u32, n: u32, o: u32) -> Weight {
-        <T as Config>::WeightInfo::execute_scheduled_instruction(f, n, o)
+        <T as Config>::WeightInfo::execute_scheduled_instruction(f, n, o).saturating_mul(2)
```

Comment:
This is was needed because the weight functions were updated and we are overcharging the user (charging each function individually), which causes the limit weight to be exceeded in a test. This is temporary and not related to this PR. I'm fixing the charges in a different PR